packages/lab/src/checkbox/Checkbox.css

```diff
@@ -1,43 +1,46 @@
 /* Styles applied to root component */
-.salt-density-medium {
-  --checkbox-fontSize: 12px;
-}
-.salt-density-touch {
-  --checkbox-fontSize: 16px;
-}
-.salt-density-low {
-  --checkbox-fontSize: 14px;
-}
-.salt-density-high {
-  --checkbox-fontSize: 11px;
-}
 
 .saltCheckbox {
   --checkbox-label-marginTop: calc(var(--salt-size-unit) * var(--checkbox-top-multiplier));
   --checkbox-space-multiplier: var(--saltCheckbox-space-multiplier, 3);
   --checkbox-top-multiplier: var(--saltCheckbox-top-multiplier, 0.75);
+  --salt-text-label-fontSize: var(--salt-text-fontSize);
+  --salt-text-label-lineHeight: var(--salt-text-lineHeight);
```

Comment:
you don't need these. the Label component will apply the text styles

Reply:
I added those overrides because the label styles didn't match the previous snapshot, but I'll remove it for now.